solc/CommandLineInterface.cpp

```diff
@@ -828,6 +830,11 @@ Allowed options)").c_str(),
 			po::value<string>()->value_name("path(s)"),
 			"Allow a given path for imports. A list of paths can be supplied by separating them with a comma."
 		)
+		(
+			g_argBasePath.c_str(),
+			po::value<string>()->value_name("path"),
+			"Allow the given path to be used as base path for relative imports."
```

Comment:
Actually ti si for absolute imports :)

Reply:
Use the given path as the root of the source tree instead of the root of the filesystem. Implies allowing access to that path for imports.